<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="cwMain">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="hlSelection">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="btnPrev">
        <property name="text">
         <string>Previous Segment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="text">
         <string>Next Segment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbxSegments"/>
      </item>
      <item>
       <widget class="QLabel" name="lblSegments">
        <property name="text">
         <string>/ 30</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlFrame">
      <item>
       <widget class="QLabel" name="lblFrame">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblGray">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlSlider">
      <item>
       <widget class="QSlider" name="slFrame">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbxFrameCurrent"/>
      </item>
      <item>
       <widget class="QLabel" name="lblFrames">
        <property name="text">
         <string>/ 30000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblCyrillic">
      <property name="text">
       <string>Cyrillic</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlCyrillic">
      <item>
       <widget class="QTextEdit" name="texCyrillic"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vlCyrillicButtons">
        <item>
         <widget class="QPushButton" name="btnOcr">
          <property name="text">
           <string>OCR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCyrillicEdit">
          <property name="text">
           <string>Edit...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblEnglish">
      <property name="text">
       <string>English</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlEnglish">
      <item>
       <widget class="QTextEdit" name="texEnglish"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vlEnglishButtons">
        <item>
         <widget class="QPushButton" name="btnTranslate">
          <property name="text">
           <string>Translate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEnglishEdit">
          <property name="text">
           <string>Edit...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project...</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>Load Project...</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionIdentify_Segments">
   <property name="text">
    <string>Identify Segments</string>
   </property>
  </action>
  <action name="actionOCR_All">
   <property name="text">
    <string>OCR All</string>
   </property>
  </action>
  <action name="actionTranslate_All">
   <property name="text">
    <string>Translate All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
